src/components/10-atoms/button/story.js

```diff
@@ -1,65 +1,65 @@
+} from '@storybook/addon-knobs';
 import './index';
 import Readme from './README.md';
 
-storiesOf('Atoms/Button', module)
-  .addParameters({
-    readme: {
-      sidebar: Readme,
-    },
-  })
-  .add('Button - default', () => '<axa-button>Default</axa-button>')
+const storyButton = storiesOf('Atoms/Button', module);
+storyButton.addDecorator(withKnobs);
+storyButton.addParameters({
+  readme: {
+    sidebar: Readme,
+  },
+});
 
-  /* Variants */
-  .add(
-    'Button - variant: red',
-    () => '<axa-button variant="red">Red</axa-button>'
-  )
-  .add(
-    'Button - variant: secondary',
-    () => '<axa-button variant="secondary">Secondary</axa-button>'
-  )
-  .add(
-    'Button - variant: inverted',
-    () => `
-    <div style="background-color: #00008f; width: 500px; height: 100px; padding-top: 30px; padding-bottom: 30px">
-      <axa-button variant='inverted'>Inverted</axa-button>
-    </div>`
-  )
+const blueBackgroundStyle = 'background-color: #00008f; padding: 10px;';
 
-  /* Disabled */
-  .add('Button - disabled', () => '<axa-button disabled>Disabled</axa-button>')
+storyButton.add('Button - default', () => {
+  const options = {
+    None: '',
+    Red: 'red',
+    Secondary: 'secondary',
+    Inverted: 'inverted',
+  };
 
-  /* Type */
-  .add(
-    'Button - type',
-    () => `
-    <div>
-      <axa-button>Button</axa-button>
-      <axa-button type="submit">Submit</axa-button>
-      <axa-button type="reset">Reset</axa-button>
-    </div>`
-  )
+  // TODO:: Move icon variants into icons and export it from there
+  const iconOptions = {
+    None: '',
+    'Arrow Right': 'arrow-right',
+    Collapse: 'collapse',
+    Document: 'document',
+    Download: 'download',
+    Email: 'email',
+    Expand: 'expand',
+    Mobile: 'mobile',
+    Phone: 'phone',
+    Search: 'search',
+    Upload: 'upload',
+  };
 
-  /* Large */
-  .add('Button - large', () => `<axa-button large>Large</axa-button>`)
+  const buttonText = text('Text', 'Click me');
+  const variants = radios('Variant', options, '');
+  const icons = select('Icon', iconOptions);
+  const motionOff = boolean('motionOff', false);
+  const disabled = boolean('disabled', false);
+  const large = boolean('large', false); // should probably be a variant.
+  const type = radios('Types', { submit: 'submit', reset: 'reset' });
 
-  /* MotionOff */
-  .add(
-    'Button - motionOff',
-    () => `
-    <div>
-        <axa-button motionoff>MotionOff</axa-button>
-        <axa-button motionoff variant="red">MotionOff + Red</axa-button>
-        <axa-button motionoff variant="secondary">MotionOff + Secondary</axa-button>
-        <axa-button motionoff variant="inverted">MotionOff + Inverted</axa-button>
-      </div>
-    </div>`
-  )
-
-  /* Icon */
-  .add(
-    'Button - icon',
-    () => `<axa-button icon="arrow-right">Icon</axa-button>`
-  );
+  return `
+  <div style='${variants === 'inverted' ? blueBackgroundStyle : ''}'>
+    <axa-button
+      ${disabled ? 'disabled' : ''}
```

Comment:
Maybe prettier to write like that <axa-button ?disabled=${disabled} ... > only works for Booleans